css/css-ui-3/text-overflow-007.html

```diff
@@ -0,0 +1,35 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: ellipsis adjacent to remaining content</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#text-overflow">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="The ellipsis is placed immediately adjacent to then end edge of the remaining inline content.">
+<style>
+.test {
+  white-space: pre;
+  font-family: ahem;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  font-size: 50px;
+  width: 2.5em;
+  line-height: 1;
+  color: green;
+}
+.red {
+  position: absolute;
+  z-index: -1;
+  width: 100px;
+  height: 100px;
+  background: red;
+  /* This red box wich will show through the gap between the letter and the ellipsis if there's one
+     but not extend past the two characters if the ellipsis is adjacent to the character as it should be.
+   */
+}
+</style>
+
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+<div class=red></div>
+<div class=test>this line should render to a "t" followed by a "…", both of which are a 1em square in ahem, whithout any gap between them.
+this line also should render to a "t" followed by a "…", both of which are a 1em square in ahem, whithout any gap between them.</div>
```

Comment:
I thought about this, and kept it the way it is because:
* If you're using AHEM, you're getting a black rectangle instead of a green square. I: 
  a. made it green
  b. added red behind to make failures more obvious
  c. doubled it so that it would be square instead of rectangular.
  a and b seemed to me to be improvements to make life easier for anyone reviewing test results, without actually making the test meaningfully more complicated. 3 may be gratuitous. I thought it reduced the chance of human error when looking at test results after having seen many squares, but I'm happy to get rid of it and add a separate reference if you think it's a net negative.
* If you're not using ahem, the code you suggested doesn't work, and you need to switch from `2.5em` to `2.5ch` and use a monospace font. You also loose the ability when looking at the test manually to tell the difference between the ellipsis being kind of close to the "A", or actually adjacent. Whether AHEM letters touch or not is clearly identifiable.

I think the comment you asked me to add about the `2.5em` width makes the test sufficiently easy to understand, and I'd like to keep it the way it is (modulo getting rid of the doubling to make things square, if you want that gone).